addons/mail/static/src/components/thread_view/thread_view_tests.js

```diff
@@ -431,6 +431,91 @@ QUnit.test('show message subject if thread is mailing channel', async function (
     );
 });
 
+QUnit.test('message list for tracking field', async function (assert) {
+    assert.expect(3);
+    await this.start();
+    this.data["mail.message"].fields.tracking_value_ids = {
+        relation: "mail.tracking.values",
+        string: "Tracking Values",
+        type: "many2many",
+    }
+    const thread = this.env.models['mail.thread'].create({
+        channel_type: 'channel',
+        id: 100,
+        members: [['insert', [
+            {
+                email: "john@example.com",
+                id: 9,
+                name: "John",
+            },
+        ]]],
+        model: 'mail.channel',
+        name: "General",
+        public: 'public',
+    });
+
+    this.data['mail.message'].records.push({
+        channel_ids: [100],
+        model: 'mail.channel',
+        res_id: 100,
+        id: 1,
+        tracking_value_ids: [{
+            changed_field: "Field 1",
+            field_type: "float",
+            id: 6,
+            new_value: 0,
+            old_value: 1,
+        }],
+    }, {
+        channel_ids: [100],
+        model: 'mail.channel',
+        res_id: 100,
+        id: 2,
+        tracking_value_ids: [{
+            changed_field: "Field 1",
+            field_type: "boolean",
+            id: 7,
+            new_value: false,
+            old_value: true,
+        }],
+    }, {
+        channel_ids: [100],
+        model: 'mail.channel',
+        res_id: 100,
+        id: 3,
+        tracking_value_ids: [{
+            changed_field: "Field 1",
+            field_type: "char",
+            id: 6,
+            new_value: '',
+            old_value: 'Name',
+        }],
+    });
+
+    const threadViewer = this.env.models['mail.thread_viewer'].create({
+        hasThreadView: true,
+        thread: [['link', thread]],
+    });
+    await this.createThreadViewComponent(threadViewer.threadView);
+    assert.strictEqual(
+        document.querySelectorAll('.o_MessageList_message .o_Message_content')[0].textContent,
+        "Field 1:10",
+        "Message content for float type field should be 'Field 1:10"
```

Comment:
this notation is really confusing without space and separator icon, in the assert comment maybe make it more visual such as:  'Field 1: 1 -> 0'
(same for others)